docs/ossm/cert-manager/README.md

```diff
@@ -30,66 +30,86 @@ You can install the cert-manager tool to manage the lifecycle of TLS certificate
 
 ### Procedure
 
-1.  Create the `istio-system` namespace:
+1.  Create the `istio-system` namespace.
 
     ```sh
     oc create namespace istio-system
     ```
 
-2.  Create the root cluster issuer:
+2.  Create the root issuer.
 
-    ```sh
-    oc apply -f - <<EOF
-    apiVersion: cert-manager.io/v1
-    kind: Issuer
-    metadata:
-      name: selfsigned
-      namespace: istio-system
-    spec:
-      selfSigned: {}
-    ---
-    apiVersion: cert-manager.io/v1
-    kind: Certificate
-    metadata:
-      name: istio-ca
-      namespace: istio-system
-    spec:
-      isCA: true
-      duration: 87600h # 10 years
-      secretName: istio-ca
-      commonName: istio-ca
-      privateKey:
-        algorithm: ECDSA
-        size: 256
-      subject:
-        organizations:
-          - cluster.local
-          - cert-manager
-      issuerRef:
-        name: selfsigned
-        kind: Issuer
-        group: cert-manager.io
-    ---
-    apiVersion: cert-manager.io/v1
-    kind: Issuer
-    metadata:
-      name: istio-ca
-      namespace: istio-system
-    spec:
-      ca:
-        secretName: istio-ca
-    EOF
-    oc wait --for=condition=Ready certificates/istio-ca -n istio-system
-    ```
-
-3.  Export the Root CA to the `cert-manager` namespace:
+    - Create the `Issuer` object as in the following example:
 
-    ```sh
-    oc get -n istio-system secret istio-ca -o jsonpath='{.data.tls\.crt}' | base64 -d > ca.pem
-    oc create secret generic -n cert-manager istio-root-ca --from-file=ca.pem=ca.pem
-    ```
+      _Example `issuer.yaml`_
 
-4.  Install istio-csr:
+      ```yaml
+      apiVersion: cert-manager.io/v1
+      kind: Issuer
+      metadata:
+        name: selfsigned
+        namespace: istio-system
+      spec:
+        selfSigned: {}
+      ---
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: istio-ca
+        namespace: istio-system
+      spec:
+        isCA: true
+        duration: 87600h # 10 years
+        secretName: istio-ca
+        commonName: istio-ca
+        privateKey:
+          algorithm: ECDSA
+          size: 256
+        subject:
+          organizations:
+            - cluster.local
+            - cert-manager
+        issuerRef:
+          name: selfsigned
+          kind: Issuer
+          group: cert-manager.io
+      ---
+      apiVersion: cert-manager.io/v1
+      kind: Issuer
+      metadata:
+        name: istio-ca
+        namespace: istio-system
+      spec:
+        ca:
+          secretName: istio-ca
+      ```
+
+    - Create the objects by using the following command.
+
+      ```sh
+      oc apply -f issuer.yaml
+      ```
+
+    - Wait for the `istio-ca` certificate to become ready.
+      ```sh
+      oc wait --for=condition=Ready certificates/istio-ca -n istio-system
+      ```
+
+3.  Export the Root CA to a local file.
```

Comment:
It's a copy of this step: https://cert-manager.io/docs/usage/istio-csr/installation/#4-export-the-root-ca-to-a-local-file but yes you're right the goal here is to copy the secret into the `cert-manager` namespace so that istio-csr can use it.